pkgs/games/20kly/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, fetchurl
+, python }:
+
+python.pkgs.buildPythonApplication rec {
+  pname = "20kly";
+  version = "1.4";
+  format = "other";
+  disabled = !(python.isPy3 or true);
```

Comment:
On python2 it's null

Reply:
`x or y` _should_ be sugar for `if x==null then y else x`, but I can't find where it's documented, if it's documented at all

Reply:
> On python2 it's null

Oh? That doesn't seem right.

And what do you want to express? Shouldn't it be `disabled = !isPy2`, since you want to disable for everything that isn't python 2?

Reply:
I don't see anybody else using `or null`. Does it make a difference if you add `isPy2` to the argument list instead of using `python.isPy2`?

Reply:
> I don't see anybody else using or null

```
git grep " or true"
git grep " or false"
```

> Shouldn't it be disabled = !isPy2, since you want to disable for everything that isn't python 2

Oops, you are right.. it should be `python.isPy3 or false` or `!(python.isPy2 or true)`... I wonder why it worked then

Reply:
> git grep " or true"

Yes but not in that context (`git grep "disabled =`)

Reply:
Did you try using `isPyX` from the argument list instead?

Reply:
Well there is `python.pkgs.callPackage` (which is used in `python-packages.nix`). But outside of `python-packages` you'd have to choose which `python` to use. Which makes the point about `disabled` kind of moot.

TLDR: I'm probably just not qualified to review a python application, sorry for the noise. I hope @domenkozar will add his opinion.